src/components/governance/GovernanceLink.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="link-wrapper">
+    <a :href="url" target="_blank" rel="noopener noreferrer">{{ title }}</a>
+    <div><astar-icon-arrow-right /></div>
```

Comment:
Why we need to wrap icon with div?

Reply:
I am using this as solution with the issue I had when arrow icons are resized based on link text size
<img width="237" alt="image" src="https://github.com/user-attachments/assets/550fa13e-3b10-46ab-a079-b7ab8bc56039">